### ISSUE TITLE
How to implement validation rules on JSON field?

### ISSUE THREAD

**Issue by xenonthong** [64robots/nova-fields#99]
Since validation support has been added, may I know how the validation rules can be applied to each individual fields?

**Comment by disitec** [comment]
Just add the rules method to the field you want to validate

```
JSON::make('BMI', [
                Number::make('Feet')->rules('required'),
                Number::make('Inches'),
                Number::make('Weight')->rules('required'),
])
```

**Comment by xenonthong** [comment]
@disitec I have 2 further questions.

1. Do I need to use 'R64\NovaFields' fields for validation to work? Or does using Nova's default field works as well?
2. What's the version of Nova that I'll have to use for the validation to work?

**Comment by disitec** [comment]
Validation should work with Nova core fields as well. I recommend you to use the latest version but I think is implemented since 0.2 or so
